libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -616,9 +616,9 @@ namespace SohImGui {
 
                 ImGui::Separator();
 
-                EnhancementCheckbox("Dpad Support on Pause and File Select", "gDpadPauseName");
-                EnhancementCheckbox("DPad Support in Ocarina and Text Choice", "gDpadOcarinaText");
-                EnhancementCheckbox("DPad Support for Browsing Shop Items", "gDpadShop");
+                EnhancementCheckbox("D-pad Support on Pause and File Select", "gDpadPauseName");
+                EnhancementCheckbox("D-Pad Support in Ocarina and Text Choice", "gDpadOcarinaText");
+                EnhancementCheckbox("D-Pad Support for Browsing Shop Items", "gDpadShop");
```

Comment:
You still have capital P on these two while it's lower-case everywhere else. Sorry, I know this is the worst kind of nitpick.

Reply:
Whoops, thought I'd changed those, thanks!